src/util/expr.h

```diff
@@ -61,6 +61,11 @@ class exprt:public irept
   {
     add(ID_type, _type);
   }
+  exprt(const irep_idt &id, const typet &type, const source_locationt &loc)
+    : exprt(id, type)
```

Comment:
make that just `source_locationt`, and then move.

Reply:
:+1: on using move